guides/storefront-guide.md

```diff
@@ -0,0 +1,821 @@
+## At a glance
+
+Mailchimp Open Commerce (MOC) is a headless commerce platform, which means that its focus is on providing a top-notch server API rather than on UI. But this doesn't mean you're completely on your own.
+
+There is a built-in user interface for shop operators and admins, which supports plugins and should work well for many shops. We refer to this as the "operator UI". If you prefer, you can build your own operator UI or adapt one that you already have.
```

Comment:
Open Commerce offers an operator UI for shop admins and operators which you can customize with plugins. You can also build your own operator UI or adapt one that you already have.